latest/docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -16,6 +16,63 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
 
 # Azure CLI 2.0 release notes
 
+## October 9, 2017
+
+Version 2.0.19
+
+### Core
+
+* Azure Stack: handle adfs authority url with a trailing slash
+
+### Appservice
+
+* webapp: added generic update with new command: 'az webapp update'
+
+### Batch
+
+* Updated to Batch SDK 4.0.0.
+* Updated --image option of VirtualMachineConfiguration to support ARM image references in addition to publish:offer:sku:version.
+* Now supports the new CLI extension model for Batch Extensions commands - support for old component model has been removed.
+
+### Batchai
+
+* Initial release of Batch AI module.
+
+### Keyvault
+
+* Fixed Key Vault authentication issue when using ADFS on Azure Stack. https://github.com/Azure/azure-cli/issues/4448
+
+### Network
+
+* `application-gateway address-pool create`: `--server` argument is not optional to allow creation of empty address pools.
+* `traffic-manager`: Updates to support latest features.
+
+### Resource
+
+* group: permit --resource-group/-g options for resource group name.
+* `account lock`: lock commands to work specifically with subscription level locks
+* `group lock`: lock commands to work specifically with group level locks
+* `resource lock`: lock command to work specifically with resource level locks
+
+### Sql
+
+* Adding support for SQL Transparent Data Encryption (TDE) and TDE with Bring Your Own Key
+* Added az sql db list-deleted command and az sql db restore --deleted-time parameter, allowing the ability to find and restore deleted databases.
+* Added az sql db op list and az sql db op cancel, allowing the ability to list and cancel in-progress operations on database.
+
+### Storage
+
+* File share snapshot
```

Comment:
No context, I need to know whether this was added, removed, changed, etc.

Reply:
It's a new feature that we now support.
I've modified this bullet point slightly.
https://azure.microsoft.com/en-us/blog/announcing-azure-files-share-snapshots-public-preview/